tests/test.edit.unit.spec.ts

```diff
@@ -442,11 +441,11 @@ test('Test case C535: Check ""Місце розташування технічн
 
     await expect(await editUnitPage.getUrl()).toContain('edit-unit');
```

Comment:
Use
```ts
await expect(editUnitPage.page).toHaveURL('edit-unit')
```
do this for all matches

Reply:
Yes, you are right that my example is not suitable, but you are wrong that you can not use this. The toHaveURL() method also can take the regular expression. So we can change it to:
```ts
await expect(editUnitPage.page).toHaveURL(/edit-unit/)

// or

await expect(editUnitPage.page).toHaveURL(RegExp('edit-unit'))
```

Reply:
Ok. I'll try

Reply:
Done